component/pyroscope/ebpf/ebpfspy/metrics/metrics.go

```diff
@@ -0,0 +1,16 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics struct {
```

Comment:
May be let's start with 2 metrics.

How many targets are active (Gauge)
Total profiling sessions succeed/failed.